script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt

```diff
@@ -171,52 +199,64 @@ The avatar in white smiles as she strides through the city.
 [sha:bc7d06d6c029d85aa33d6ef4fd9113a189d679b2]{
 -- Page 283, Offset 39189.
 -- その足取りは軽く、嬉しくてたまらないと全身で謳っている。
-Her pace is light, and her entire body celebrates with irrepressible happiness.
+Her footsteps are light,
+ and her whole body radiates an almost unbearable happiness.
 }
 [sha:406b4fb431a819046f2ae0cdcd8f98cbccfa567a]{
 -- Page 284, Offset 39190.
 -- 一方、シエルの肉体は限界を迎えていた。
-Meanwhile, Ciel's body is reaching its limit.
+Ciel's body, meanwhile, has reached its limit.
 }
 [sha:865458363841e7ad05c12c43b0f4ab3b764d578e]{
 -- Page 284, Offset 39191.
 -- 魔力はまだ湧き上がるものの、肉体が悲鳴を上げている。
-Although her magical energy continues to surge, her body continues to scream.
+Even though she's still bursting with magical energy,
+ her body is screaming in protest.
 }
 [sha:5b7eb47b4c51caa6b2f4dbe33c9a9704fb672110]{
 -- Page 284, Offset 39192.
 -- 両脚の骨、肋骨、１トンの銃器を握る左腕は骨折と復元を秒単位で繰り返している。
-The bones in her legs, ribs, and the left arm carrying the one-ton firearm, repeat the same process of being fractured and restored in a matter of seconds.
+The bones in her legs, her ribs,
+ and the left arm with which she carries her one-ton firearm
+ undergo the same process of being broken and restored over and over again with each passing second.
 }
 [sha:086ca7dfb0bdf2badc5d57432a04fae671e177d8]{
 -- Page 285, Offset 39193.
 -- 不死の<法則|のろい>が体を復元するが、その復元より早く骨が砕かれ、筋肉が千切れ、内臓が破裂する。
-Although the curse of immortality restores her body, her bones shatters, her muscles tears, and her organs rupture faster then they could be regenerated.
+But even though the curse of immortality restores her body,
+ her bones shatter,
+ her muscles tear,
+ and her organs rupture faster than they can be restored.
 }
 [sha:4f2bd4a8978914e91f5bd4ebf143ce469b8b9c41]{
 -- Page 285, Offset 39194.
 -- 見えざる衝撃と、それを躱す為の回避運動が、彼女の内部を破壊し続ける。
-The invisible assaults, and the evasive maneuvers to avoid such assaults, continue to destroy her insides.
+The invisible impacts,
+ and the evasive maneuvers she performs to avoid them,
+ continue to pulverize her insides.
```

Comment:
Unseeable weirdly obtuse, and the 'combined with' doesn't really play nice with the rest of the sentence

Reply:
I'm not sure how "unseeable" is obtuse, I think a 5 year old could understand it. I was trying to emulate 見えざる, which is more 大和言葉より, etc. It's not a big deal, but I don't really see why it would be obtuse.